docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+
+- Connection to VCS repository where MLFlow project for wine classification is located
+- Connection to wine-quality.csv file in one of supported object storage
+- Connection to docker repo where out trained and packed model will be stored
+
+Let's create these connections. Legion examples repository already contains payload templates so we will use it
+
+Let's create connection to VCS repository that contains project code. Because legion-examples repository already contains required project
+we just create connection that points to this Git repository.
+
+.. code-block:: console
+
+    $ cat ./legion/vcs_connection.legion.yaml
+
+    kind: Connection
+    id: legion-examples
+    spec:
+      type: git
+      uri: git@github.com:legion-platform/legion-examples.git
+      reference: origin/master
+      publicKey: <paste here your public github ssh key>
+      keySecret: <paste here your key github ssh key>
+      description: Git repository with legion-examples
+      webUILink: https://github.com/legion-platform/legion-examples
+
+So you need to replace <...> variables in this template with actual values and then create connection:
+
+`Using legionctl`
+
+.. code-block:: console
+
+    $ legionctl conn create -f ./legion/vcs_connection.legion.yaml
+
+`Using jupyterlab`
+
+1. Open jupyterlab
+2. Open cloned repo, and then folder with project
+3. Select file ``./legion/vcs_connection.legion.yaml`` and in context menu press ``submit`` button
+
+Then you need to create connection that will points to our wine-quality.csv file
+
+We will use ``gcs`` connection type for this
+
+.. code-block:: console
+
+    $ cat ./legion/wine_connection.legion.yaml
+
+    kind: Connection
+    id: wine
+    spec:
+      type: gcs
+      uri: gs://<paste your bucket address here>/data/wine-quality.csv
+      region: <past region here>
+      keySecret: <paste key secret here>
+      description: Wine dataset
+
+Apply this payload absolutely the same way as we did for VCS connection.
+
+Of course we should provide wine-quality.csv file for connection listed above
+
+.. code-block:: console
+
+    $ cat gsutil cp ./data/wine-quality.csv gs://gke-legion-dev01-data-store/data/
+
+
+Then create connection to docker registry. Legion Platform will use this connection for uploading packed models
+
+.. code-block:: console
+
+    $ cat ./legion/docker_connection.legion.yaml
+
+    kind: Connection
+    id: docker-ci
+    spec:
+      type: docker
+      uri: <past uri of your registry here>  # for example: gcr.io/or2-msq-epmd-legn-t1iylu/gke-legion-ds
+      username: <paste your username here>
+      password: <paste your password here>
+      description: Docker repository for model packaging
+      webUILink: <paste webui link>
+
+Apply this payload absolutely the same way as we did for VCS connection or wine dataset connection
+
+Let's check all created connections
+
+.. code-block:: console
+
+    $ legionctl conn get | grep -e id: -e type: -e description
```

Comment:
Do we have simple view (I mean columnar?)
@mcdoker18 ?

Reply:
I guess using grep in tutorial is not a good sign :)

Reply:
```bash
legionctl conn get -o 'jsonpath=$[*].id'
legionctl conn get -o table
```
But I think the grep is good.